nemo_text_processing/text_normalization/graph_utils.py

```diff
@@ -0,0 +1,208 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Can this file be shared between ITN and TN?

Reply:
I tried and its not trivial, imports mess up references. Can i try this in another PR?

Reply:
Ok, let's leave it for a separate PR